src/util/async/AnyStrand.hpp

```diff
@@ -165,7 +167,7 @@ class AnyStrand {
     };
 
 private:
-    std::unique_ptr<Concept> pimpl_;
+    std::shared_ptr<Concept> pimpl_;
```

Comment:
Why is this shared now?

Reply:
The overhead is not that much but it will allow us to copy these type-erased wrappers without a care in the world, just like we do with yield_context 👍 
Consider that this is only for AnyExecutionContext and AnyStrand, not for AnyOperation. This is because conceptually there will not be much copying of the ExecutionContext/Strand around (outside of DI). Operation on the other hand might be stored somewhere and removed later which can lead to shared state manipulation and that can lead to contention on the atomic inside if multiple threads are involved.